pkg/controller/convert.go

```diff
@@ -52,6 +53,7 @@ func IdentifyResourcePresence(fs *afero.Afero, outputDir string, hzFetcher Hoste
 		hasArgoCD:                             directoryTester(fs, outputDir, config.DefaultArgoCDBaseDir),
 		hasDelegatedHostedZoneNameservers:     hz != nil && hz.IsDelegated,
 		hasDelegatedHostedZoneNameserversTest: false,
+		hasUsers:                              false, // this means we will always check? needs to be changed?
```

Comment:
This means it will always try to create the users in declaration.users. If the create users command is idempotent, this will work for now

Reply:
Good, that is what I thought, and it is idempotent 👍🏽